tensorflow/contrib/periodic_resample/ops/array_ops.cc

```diff
@@ -26,7 +26,42 @@ REGISTER_OP("PeriodicResample")
     .Input("values: T")
     .Attr("shape: shape")
     .Output("output: T")
-    .SetShapeFn(shape_inference::ExplicitShape)
+    .SetShapeFn([](shape_inference::InferenceContext* c) {
```

Comment:
Thanks for adding a shape inference function. Can you please add a unit test for periodic_resample that takes a placeholder without specified shape to make sure everything works as intended?

Reply:
Added testPeriodicResampleShapeInference test

Reply:
Sorry, I was not clear. The shape inference tests go in array_ops_test.cc. It doesn't hurt to have the Python test also.

Reply:
Sorry, I am confused a bit. Test I've added testPeriodicResampleShapeInference - placed in Python file /tensorflow/contrib/periodic_resample/python/kernel_tests/periodic_resample_op_test.py
You mean, I should add file named array_ops_test.cc to test this functionality in C++ also, correct? 

For file you mentioned I've found /tensorflow/core/ops/array_ops_test.cc , but not sure - is adding C++ test for some operation from contrib to test inside /tensorflow/core is really what you meant?

Reply:
@rmlarsen could you explain remark above, please? Hope it will  not take  much time. Thank you in advance!

Reply:
Not sure whether I've understand you correctly. I've added array_op_test.cc to the periodic_resample ops dir, when placed test for shape inference functionality. Is that what you meant or I misunderstand you?